src/aspire/ctf/ctf_estimator.py

```diff
@@ -291,7 +291,7 @@ def background_subtract_1d(
         Estimate and subtract the background from the power spectrum
 
         :param amplitude_spectrum: Estimated power spectrum
-        :param linprog_method: Method passed to linear progam solver (scipy.optimize.linprog).  Defaults to 'interior-point'.
+        :param linprog_method: Method passed to linear progam solver (scipy.optimize.linprog).
```

Comment:
"program"

Reply:
changed.  Note why I was updating this line..

Reply:
Noted. Yeah, sorry for missing that!

Reply:
no good deed something something